public/scripts/aboutView.js

```diff
@@ -0,0 +1,31 @@
+(function(module){
+  var aboutView = {};
+
+  aboutView.getSections = function(path, callback){
```

Comment:
It looks like this `.getSections` could be consolidated inside the `aboutView.index()`. It's just doing a little validation for you, which could be handled inside the primary route callback.